scripts/west_commands/runners/bossac.py

```diff
@@ -9,14 +9,18 @@
 import pickle
 import platform
 import subprocess
-import sys
 
 from runners.core import ZephyrBinaryRunner, RunnerCaps, BuildConfiguration
-from zephyr_ext_common import ZEPHYR_SCRIPTS
 
 # This is needed to load edt.pickle files.
-sys.path.append(str(ZEPHYR_SCRIPTS / 'dts'))
-import edtlib  # pylint: disable=unused-import
+try:
+    import edtlib  # pylint: disable=unused-import
+    MISSING_EDTLIB = False
```

Comment:
Nit from the UFADN[*] department:
```suggestion
    FOUND_EDTLIB = True
```
[*] United Front Against Double Negations.